### ISSUE TITLE
Console history popup not announced by screen reader

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6168]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.815
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
With a screen reader running, and screen-reader support enabled, type <kbd>Cmd+Up Arrow</kbd> (Mac) or <kbd>Ctrl+Up Arrow</kbd> Windows/Linux in the console to show command history popup.

### Describe the problem in detail
No announcements made by screen reader about the popup, or if you move around the popup with arrow keys.

### Describe the behavior you expected
Should announce the popup in a way that makes it usable for screen reader users.

There is a workaround in that you can hit just the up/down arrow keys to navigate the history in-place, and it will be announced.

**Comment by gtritchie** [comment]
Important for screen reader usage.
